### PR TITLE
Completed TC05 31/8/2021

### PR DESCRIPTION
**Pull request by HoanVanHuynh**
System shows message when user enters wrong password several times

### CODE REVIEW

File: src/test/java/tests/LoginTests.java [hunk 1]
```diff
@@ -98,4 +95,21 @@ public void tc04_UserIsRedirectedToBookTicketPage() {
         LogHelper.info("Verify that book ticket page displays with Book ticket form opens");
         Assert.assertEquals(actualTitle, "Book ticket form", "Title of Book ticket form is displayed incorrectly as expected");
     }
+
+    @Test(description = "System shows message when user enters wrong password several times", groups = {"g1"})
+    public void tc05_SystemShowsMessage() {
+        String invalidPassword = "11122233345";
+
+        LogHelper.info("Log in 4 times with valid username but invalid password");
+        for (int i = 0; i < 4; i++) {
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java]: Please create a function for this loop instead of using it directly on test

Comment (HoanVanHuynh) [reply src/test/java/tests/LoginTests.java]: please let me know where to put that loop function, should I create another class, or folder to put it?

Comment (MinhQuan119) [reply src/test/java/tests/LoginTests.java]: The feature is still login, so you can put it in LoginPage

Comment (HoanVanHuynh) [reply src/test/java/tests/LoginTests.java]: you're so smart, thank you so much

File: src/main/java/page_objects/LoginPage.java [hunk 2]
```diff
@@ -59,4 +59,10 @@ public String getErrorMessageAtTop() {
     public String getTopicContentAtTop() {
         return this.getLblTopicContentAtTop().getText();
     }
+
+    public void loopLogin(int numberOfTimes, String username, String password) {
```
Comment (MinhQuan119) [inline src/main/java/page_objects/LoginPage.java]: Check english grammar. Besides, it should be a verb

**Review by tanmle — APPROVED** [review]

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**